service-deviceregistry/requirements.in

```diff
@@ -0,0 +1,8 @@
+Django==4.2.1,<5.0
```

Comment:
This may pin Django==4.2.1, change to
`Django>=4.2.1,<5.0`

Reply:
done